core/src/parser.rs

```diff
@@ -74,6 +74,10 @@ pub enum ParseError {
     SerdeTagRequired { enum_ident: String },
     #[error("serde content attribute needs to be specified for algebraic enum {enum_ident}. e.g. #[serde(tag = \"type\", content = \"content\")]")]
     SerdeContentRequired { enum_ident: String },
+    #[error("the expression assigned to this constant variable is not a numeric literal")]
+    RustConstExprInvalid,
+    #[error("you cannot use typeshare on a constant that is not a numeric literal")]
+    RustConstTypeInvalid,
```

Comment:
Can we document these limitations in an issue? We want to make it as easy as possible for a contributor to pick this up, if they want to expand typeshare.

Reply:
It should actually be two issues: 
* one about adding constants other than int types
* one about adding constants for Swift, Scala and Kotlin

Reply:
Issues created: #228, #229